index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="A precall workflow application using the Daily client SDK, Daily Precall React and Daily React"
```

Comment:
Daily Precall React isn't a thin anymore, right? So probably worth removing from the meta.